src/packages/@ncigdc/components/Header/styles.scss

```diff
@@ -142,6 +142,7 @@
     > li > button,
     .header-link {
       align-items: center;
+      background-color: #fff;
```

Comment:
Strange, I've already added this change, why is Github not picking that up? 🤔

![image](https://user-images.githubusercontent.com/2107110/81601028-78d8af00-9398-11ea-822d-a40dfebd1734.png)